Layout.hs.patch

```diff
@@ -0,0 +1,10 @@
+--- Web/View/Layout.hs   2022-01-15 10:05:24.712496967 +0100
++++ Layout.hs  2022-01-15 10:06:30.321569568 +0100
+@@ -54,6 +54,7 @@
+         <script src={assetPath "/helpers.js"}></script>
+         <script src={assetPath "/ihp-auto-refresh.js"}></script>
+         <script src={assetPath "/app.js"}></script>
++        <script src={assetPath "/miso/index.js"} type="module"></script>
```

Comment:
These patch files of course mean that this repo isn't actually runnable. I developed these changes in a repo created by `ihp-new` and then copied them here. If we make further changes here, is there a good way to test them, other than building a modified version of `ihp-new` which can point to this branch?

Reply:
Currently there is no better way to test this, the boilerplate is not changed that often typically